showcase/plot/mixed-stacked-chart.js

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2016 - 2017 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+import React from 'react';
+import ShowcaseButton from '../showcase-components/showcase-button';
+import {
+  XYPlot,
+  XAxis,
+  YAxis,
+  VerticalGridLines,
+  HorizontalGridLines,
+  VerticalBarSeries,
+  VerticalBarSeriesCanvas,
+  LineSeries
+} from 'index';
+
+export default class Example extends React.Component {
```

Comment:
This is a good example! I like that we are checking that stacking works in canvas and svg. Would you mind writing a test against this example so that we don't back slide?

Reply:
your example exposes a kinda funky ambiguity 
![ex](https://user-images.githubusercontent.com/6854312/36052790-bcb4f178-0db4-11e8-97f0-fe140ed91ff3.gif)

The type of component that you are varying correctly switches between canvas and svg, but the other component (the line) remains an svg node which is weird! react-vis is clearly behaving as it should, however it is not spelled out in the docs the why and way this behaviour should be! 

nice! good edge pushing.